packages/loopring_v3/contracts/core/impl/libtransactions/WithdrawTransaction.sol

```diff
@@ -33,7 +33,7 @@ library WithdrawTransaction
     using ExchangeWithdrawals  for ExchangeData.State;
 
     bytes32 constant public WITHDRAWAL_TYPEHASH = keccak256(
-        "Withdrawal(address owner,uint32 accountID,uint16 tokenID,uint256 amount,uint16 feeTokenID,uint256 fee,address to,bytes32 dataHash,uint24 minGas,uint32 validUntil,uint32 nonce)"
+        "Withdrawal(address owner,uint32 accountID,uint16 tokenID,uint256 amount,uint16 feeTokenID,uint256 fee,address to,bytes data,uint minGas,uint32 validUntil,uint32 nonce)"
```

Comment:
the order doesn't match the struct definition.

Reply:
The struct tries to make it clear that part of the data is from the DA and the other part of the data is from the aux data. I thought it would be clearer this way, otherwise the parameters from the 2 different sources are mixed. `WITHDRAWAL_TYPEHASH` also does not contain all the data like `dataHash`, so they are not completely the same either way.

Reply:
I agree, i just want to see them as in sync as possible to improve readability.

Reply:
The main problem is I think the dynamically sized bytes. To decode those we'd basically have to implement our own simple ABI and decoder for it, at which point it seems to make more sense to just use the built in stuff when possible.